app/src/net/fieldwire/models/response/AccountUser.java

```diff
@@ -0,0 +1,33 @@
+package net.fieldwire.models.response;
+
+import java.util.Date;
+
+public class AccountUser extends BaseModel<Integer> {
+    public String firstName;
+    public String lastName;
+    public String accountRole;
+    public String email;
+    public String company;
+    public String phoneNumber;
+    public String photoUrl;
+    public Date currentSignInAt;
+    public int invitedById;
+
+    @Override
+    public String toString() {
+        return "User{" +
```

Comment:
If I've read correctly, we never log `AccountUser` and `ProjectUser` directly from the sample code, but I guess this is for convenience if our customers want to debug these entities, right?

Reply:
- True for `AccountUser` => added it just for the sake of completeness
- `ProjectUser` => this one gets logged as part of the regional flow highlighted below:

```java
ProjectUser user = projectUsersResponse.users.get(0);
log_info("Invited user to project", user.toString());
```